docs/troubleshooting/polling-issues.md

```diff
@@ -42,11 +42,13 @@ If you put in only the IP address (for example, `127.0.0.1`), then errors will b
 
 ## Walking a device takes too much time
 
-See [Configure small walk profile](../configuration/configuring-profiles/#walk-profile) to enable the small walk functionality.
+See [Configure small walk profile](../../configuration/configuring-profiles/#walk-profile) to enable the small walk 
```

Comment:
It seems here we have `../../configuration` but on the bottom (line 129) `../configuration`.
As I understood one of that is link, another one is md file. Can we use `configuring-profiles.md#walk-profile` ?

Reply:
<img width="1271" alt="Screenshot 2024-10-07 at 14 35 51" src="https://github.com/user-attachments/assets/04241e7d-adc3-4569-84a4-14ca7caef526">
mkdocs also giving such warning